src/Service/Accounts/AccountsService.php

```diff
@@ -225,7 +227,7 @@ public function verifyToken($idToken)
         return $this->getClient()->post(
             '/v1/shop/token/verify',
             [
-//                Request::HEADERS => $this->getHeaders(),
+                //                Request::HEADERS => $this->getHeaders(),
```

Comment:
again vscode tabs ?

Reply:
You don't really like vscode :heart:

Reply:
besides, neither comment indention is good